hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/listener/ClientNonSmartListenerService.java

```diff
@@ -184,7 +185,7 @@ public void run() {
                             return Collections.EMPTY_LIST;
                         }
                         LinkedList<ClientEventRegistration> activeRegistrations = new LinkedList<ClientEventRegistration>();
-                        if (getEventHandler(registration.getCallId()) != null) {
+                        if (registration.isActive()) {
```

Comment:
event handler is registered before invocation is send. I need a value to check that is updated after invocation is send and successful.